content/en/docs/concepts/scheduling-eviction/kube-scheduler.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Kubernetes Scheduler
 content_type: concept
-weight: 10
+weight: 60
```

Comment:
I'm not sure about this @gm7y8. The kube-scheduler feels to me like it _should_ be front and center in any topic about scheduling.

I'm open to being persuaded it should change - how did you pick the revised weight figure?

Reply:
I picked this weight from reference commit. sure i can move it to front and center.